client/src/relatedProducts/RelatedProductsView/RelatedProducts.jsx

```diff
@@ -22,12 +22,15 @@ export default class RelatedProducts extends React.Component {
       modalShow: false,
       clickedProductInfo: {},
       modifiedCurrent: {},
-      features: []
+      features: [],
+      displayedProductsIndices: [0, 1, 2]
     }
     this.handleAddToOutfit = this.handleAddToOutfit.bind(this);
     this.handleRemoveFromOutfit = this.handleRemoveFromOutfit.bind(this);
     this.handleCompareItems = this.handleCompareItems.bind(this);
     this.closeModal = this.closeModal.bind(this);
+    this.handlePrevClick = this.handlePrevClick.bind(this);
```

Comment:
not anything that needs to even be done, but could these two functions be combined into one handler to save space?  theoretically the clicks could be separated by a condition which decides which state to update maybe?